package.json

```diff
@@ -49,6 +49,7 @@
     "lodash.isempty": "^4.4.0",
     "mailgen": "^2.0.7",
     "methods": "^1.1.2",
+    "moment": "^2.24.0",
```

Comment:
Why did you install this ?

Reply:
I am working with time stamp

Reply:
There are much smaller libraries available. Did you care to check the size of the library on your bundle